### PR TITLE
svc: Minor enum changes

### PR DESCRIPTION
**Pull request by lioncash**

### CODE REVIEW

**Comment by JayFoxRox** [comment]
LGTM

File: src/core/hle/svc.h [hunk 1]
```diff
@@ -20,20 +20,10 @@ struct PageInfo {
...
-    ARBITRATIONTYPE_SIGNAL,
-    ARBITRATIONTYPE_WAIT_IF_LESS_THAN,
-    ARBITRATIONTYPE_DECREMENT_AND_WAIT_IF_LESS_THAN,
-    ARBITRATIONTYPE_WAIT_IF_LESS_THAN_WITH_TIMEOUT,
-    ARBITRATIONTYPE_DECREMENT_AND_WAIT_IF_LESS_THAN_WITH_TIMEOUT,
-    ARBITRATIONTYPE_MAX_BIT = (1u << 31)
+enum class ResetType {
+    OneShot,
+    Sticky,
+    Pulse,
```
Comment (yuriks) [inline src/core/hle/svc.h]: This enum should be moved to event.h, to match with the ArbitrationType one.

Comment (lioncash) [inline src/core/hle/svc.h]: Done

**Comment by yuriks** [comment]
LGTM

**Comment by Subv** [comment]
LGTM

**Pull request merged**